test/controllers/user.test.ts

```diff
@@ -32,4 +32,25 @@ describe('User class', () => {
             ).toThrowError();
         });
     });
+    describe('generateKeywords()', () => {
```

Comment:
Your describe statement is about several functions, not just generateKeyword(). Either create a describe statement for all functions, or remove it and keep tests together with the other user tests